puppeteer-pdf.js

```diff
@@ -1,130 +1,111 @@
 #!/usr/bin/env node
+import {createRequire} from 'module';const require=createRequire(import.meta.url);
+var __require = /* @__PURE__ */ ((x) => typeof require !== "undefined" ? require : typeof Proxy !== "undefined" ? new Proxy(x, {
+  get: (a, b) => (typeof require !== "undefined" ? require : a)[b]
+}) : x)(function(x) {
+  if (typeof require !== "undefined")
+    return require.apply(this, arguments);
+  throw new Error('Dynamic require of "' + x + '" is not supported');
+});
```

Comment:
That could use some prettifying I believe 😅

Reply:
@ahallora This is the executable that is fired so it doesn't matter what's inside :D